pkg/sql/logictest/testdata/logic_test/fk

```diff
@@ -2715,9 +2715,16 @@ statement ok
 CREATE TABLE child_duplicate_cols (id INT, parent_id INT, PRIMARY KEY (id))
 
 # The fk constraint is invalid because it has duplicate columns, so automatically adding the index fails
+# Unfortunately, the error message is slightly different between legacy and declarative schema changer
+# so we have the same statement twice here.
```

Comment:
Should we leave a TODO to fix this? Seems like the fully qualified name is a bit over the top in this case.

Reply:
Use a regexp that matches both schema changer implementations.